src/xo-app/backup/restore/index.js

```diff
@@ -0,0 +1,215 @@
+import ActionButton from 'action-button'
+import find from 'lodash/find'
+import forEach from 'lodash/forEach'
+import Icon from 'icon'
+import Link from 'react-router/lib/Link'
+import map from 'lodash/map'
+import orderBy from 'lodash/orderBy'
+import React, { Component } from 'react'
+import size from 'lodash/size'
+import { connectStore, formatSize } from 'utils'
+import { createGetObject, createGetObjectsOfType } from 'selectors'
+import { DropdownButton, MenuItem } from 'react-bootstrap-4/lib'
+import { info, error } from 'notification'
+import { FormattedDate } from 'react-intl'
+import { Row, Col } from 'grid'
+
+import {
+  importBackup,
+  importDeltaBackup,
+  listRemote,
+  subscribeRemotes
+} from 'xo'
+
+const deltaBuilder = (backups, uuid, name, tag, value) => {
```

Comment:
AFAIU, this function groups backup files by VM id (avoid the term uuid), then by name and finally by tag.